config/jobs/cert-manager/istio-csr/cert-manager-istio-csr-presubmits.yaml

```diff
@@ -282,3 +282,57 @@ presubmits:
         options:
           - name: ndots
             value: "1"
+
+  # kind based istio-csr e2e job for Kubernetes v1.21, istio v1.11
+  - name: pull-cert-manager-istio-csr-k8s-v1-21-istio-v1-11
+    context: pull-cert-manager-istio-csr-k8s-v1-21-istio-v1-11
+    # Match everything except PRs that only touch docs/
+    always_run: true
+    optional: false
+    max_concurrency: 8
+    agent: kubernetes
+    decorate: true
+    branches:
+    - ^main$
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: eu.gcr.io/jetstack-build-infra-images/golang-dind:20210331-a8721c1-1.16
```

Comment:
There is a Go 1.17 image now https://console.cloud.google.com/gcr/images/jetstack-build-infra-images/eu/golang-dind?authuser=2&project=jetstack-build-infra-images&tab=vulnz

Reply:
Thanks, I went ahead and updated all the images in the file.